lua/entities/gmod_wire_expression2/base/preprocessor.lua

```diff
@@ -200,7 +200,7 @@ local function handleIO(name)
 
 		for i, key in ipairs(retval[1]) do
 			if ports[3][key] then
-				self:Error("Directive (@" .. name .. ") contains multiple definitions of the same variable", columns[i])
+				self:Warning("Directive (@" .. name .. ") contains multiple definitions of the same variable", columns[i])
```

Comment:
Should this account for type mismatches or defining inputs as persist?

Reply:
Indeed forgot to account for type mismatches